ConsentViewController/Classes/MessageWebViewController.swift

```diff
@@ -188,7 +190,7 @@ class MessageWebViewController: GDPRMessageViewController, WKUIDelegate, WKNavig
     }
 
     override func loadMessage(fromUrl url: URL) {
-        load(url: url)
+        load(url: URL(string: "\(url.absoluteString)&consentLanguage=\(messageLanguage.rawValue)")!)
```

Comment:
We should avoid building the query params with string interpolation. Furthermore, we just introduced a force unwrap.

Reply:
Added extension for URL to append the query params.